library/styleguideComponents/RibbonRenderer/index.js

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import GitHubButton from 'react-github-btn';
 import StackoverflowIcon from './stackoverflowIcon';
 import YouTubeIcon from './youtubeIcon';
-import GlobalSearch from '../GlobalSearch';
```

Comment:
if GlobalSearch will not be used more, then go and clean up it and remove all not used